src/main/java/frc/robot/Constants.java

```diff
@@ -365,9 +367,10 @@ public static final class ScoringConstants {
 
         public static final double aimMaxAngleRadians = Math.PI / 2;
 
-        public static final double timeToPutAimDown = 2;
+        // NOTE: This is unitless (not in seconds)
+        public static final double timeToPutAimDown = 0.5;
```

Comment:
wait, why would this be unitless? It should be in seconds lol

Reply:
The way I initially did velocity was with weird scaling and the units would not work traditionally. It now works and I can change this to be seconds. Sorry